### PR TITLE
Ibp 4603 add release note v18

### PR DESCRIPTION
**Pull request by lik-83**

### CODE REVIEW

File: src/main/resources/liquibase/workbench_changelog/17_4_0.xml [hunk 1]
```diff
@@ -240,5 +240,29 @@
 		</sql>
 	</changeSet>
 
+	<changeSet author="lkovacic" id="v17.4.0-7">
```
Comment (clarysabel) [inline src/main/resources/liquibase/workbench_changelog/17_4_0.xml:243]: @lik-83 This changeset needs to be moved to a new file: 17_4_1.xml

File: src/main/resources/liquibase/workbench_changelog/17_4_0.xml [hunk 2]
```diff
@@ -240,5 +240,29 @@
...
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="release_note" columnName="file_name"/>
+			</not>
+		</preConditions>
+        <comment>Add 'field_name' column to 'release_note' table</comment>
```
Comment (clarysabel) [inline src/main/resources/liquibase/workbench_changelog/17_4_0.xml:249]: typo field by file

File: src/main/resources/liquibase/workbench_changelog/17_4_0.xml [hunk 3]
```diff
@@ -240,5 +240,29 @@
...
+		<sql dbms="mysql" splitStatements="true">
+			alter table release_note add file_name varchar(255) not null;
+		</sql>
+	</changeSet>
+
+	<changeSet author="lkovacic" id="v17.4.0-8">
```
Comment (clarysabel) [inline src/main/resources/liquibase/workbench_changelog/17_4_0.xml:255]: move to the new file

File: src/main/resources/liquibase/workbench_changelog/17_4_0.xml (hunk 2 above)
Comment (lik-83) [reply src/main/resources/liquibase/workbench_changelog/17_4_0.xml]: Done

File: src/main/resources/liquibase/workbench_changelog/17_4_0.xml (hunk 3 above)
Comment (lik-83) [reply src/main/resources/liquibase/workbench_changelog/17_4_0.xml]: Done

**Pull request merged**